worker/src/task/3-audit.ts

```
@@ -2,29 +2,27 @@ import { getOrcaClient } from "@_koii/task-manager/extensions";
-export async function audit(cid: string, roundNumber: number, submitterKey: string): Promise<boolean | void> {
-  /**
-   * Audit a submission
-   * This function should return true if the submission is correct, false otherwise
-   * The default implementation retrieves the proofs from IPFS
-   * and sends them to your container for auditing
-   */
 
+const TIMEOUT_MS = 180000; // 3 minutes in milliseconds
+const MAX_RETRIES = 3;
+
+async function auditWithTimeout(cid: string, roundNumber: number, submitterKey: string): Promise<boolean | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)